Feature/edit from filelist

klaseskilson opened the pull request:
This allows files to be edited and deleted from the file list, without reloading the whole page.

![skarmavbild 2015-05-22 kl 15 38 17](https://cloud.githubusercontent.com/assets/1674076/7771609/d6341a38-0098-11e5-8d81-4072a3e59dea.png)

klaseskilson committed:
Move cols in filelist, make jseditor better

klaseskilson committed:
Name and tags can be edited

klaseskilson committed:
Files can be edited from file list :partly_sunny:

klaseskilson committed:
Preserve old file name when editing file

klaseskilson committed:
Files can be deleted from file list

klaseskilson committed:
Improve width control of update file button

klaseskilson committed:
Merge branch 'develop' of github.com:tkomstadius/bruse into feature/edit-from-filelist

danielronnkvist commented on app/controllers/files/files_controller.rb:
```diff
@@ -63,7 +65,7 @@ def index
   #
   def update
-    @file = BruseFile.find(params[:bruse_file][:id])
+    @file = current_user.bruse_files.find(params[:bruse_file][:id])
```
nice! :zap:

klaseskilson replied:
:ok_hand:

danielronnkvist commented:
Cool! Ready for merge :pineapple:

ronjagrosz commented:
![intebra](https://cloud.githubusercontent.com/assets/10630598/7773156/ef71a9ca-00a2-11e5-9918-b53093924499.png)
:cry:

klaseskilson commented:
Hmm. Not too good looking, no. 🙈 I'll take a look at it.

klaseskilson committed:
Quick fix for file tools breaking row

Also, clarify some instructions and comments and make sure links
aren't appearing as previewable.

klaseskilson commented:
The issues regarding row-breaking icons in the tools area is that tables are hard to control. I'm not sure how we should handle this, but one idea could be to move everything from the `table`-based structure we have today to a `div`-based structure. I think that could be done in an other PR, but it could help us get rid of this unreliable table we have today.

ronjagrosz commented:
looks good!

danielronnkvist commented:
Looks good, will merge! Regarding migrating to a `div` based structure I think that has a low priority until Friday.
